### ISSUE TITLE
Enable GitHub Discussions on this repository

### ISSUE THREAD

**Issue by byaruhaf** [razeware/emitron-iOS#671]
Could we [enable GitHub discussions](https://docs.github.com/en/discussions/quickstart) on this repository, Discussions are great for questions and ideas that require team communication to make a decision, while Issues are defined pieces of work.

Thus issues like #670 & #672 would be more appropriate in a discussions section rather than an issues section.

**Comment by JessyCatterwaul** [comment]
Good idea, @byaruhaf. It requires a template "to align with the resources and tone you want to set for your community". Do you have any good examples as inspiration?
